packages/lab/src/slider/internal/SliderThumb.tsx

```diff
@@ -1,82 +1,163 @@
-import { Label, makePrefixer } from "@salt-ds/core";
-import { clsx } from "clsx";
-import { type ComponentPropsWithoutRef, useState } from "react";
-import type { ActiveThumbIndex, ThumbIndex } from "../types";
-import { useSliderContext } from "./SliderContext";
-import { useKeyDownThumb } from "./useKeyDownThumb";
-import { getPercentage } from "./utils";
+import { makePrefixer, useId } from "@salt-ds/core";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { useWindow } from "@salt-ds/window";
+import clsx from "clsx";
+import {
+  type ChangeEvent,
+  type HTMLAttributes,
+  useEffect,
+  useRef,
+  useState,
+} from "react";
+import sliderThumbCss from "./SliderThumb.css";
+import { SliderTooltip } from "./SliderTooltip";
 
 const withBaseName = makePrefixer("saltSliderThumb");
 
-export interface SliderThumbProps extends ComponentPropsWithoutRef<"div"> {
-  index: ThumbIndex;
-  activeThumb: ActiveThumbIndex;
-  setActiveThumb: (index: ActiveThumbIndex) => void;
+interface SliderThumbProps
+  extends Omit<
+    HTMLAttributes<HTMLInputElement>,
```

Comment:
```suggestion
    ComponentPropsWithoutRef<"input"> ,
```

Will cover more types